src/microbe_stage/AvailableUpgrade.cs

```diff
@@ -75,6 +83,19 @@ public void Resolve()
     {
         if (!string.IsNullOrEmpty(IconPath))
             LoadedIcon = GD.Load<Texture2D>(IconPath);
+
+        // Preload the scene for instantiating in microbes
+        // TODO: switch this to only load when loading the microbe stage to not load this in the future when we have
+        // playable stages that don't need these graphics
+        if (!string.IsNullOrEmpty(graphics.ScenePath))
+        {
+            loadedSceneData.LoadFrom(graphics);
+        }
+    }
+
+    public LoadedSceneWithModelInfo? TryGetGraphicsScene()
```

Comment:
This cannot return a nullable value as that return type is a struct, which will cause boxing in this case. That means this has to be written like the `TryGet` method I wrote for the OrganelleDefinition.